src/main/java/io/github/aquerr/steamwebapiclient/SteamWebApiClient.java

```diff
@@ -70,18 +70,18 @@ public SteamWebApiClient(String apiKey, HttpClient httpClient, ObjectMapper obje
     }
 
     private SteamWebApiClient(SteamHttpClient steamHttpClient) {
-        this.workshopWebApiClient = new SteamWorkshopWebApiClient(steamHttpClient);
+        this.workshopWebApiClient = new SteamPublishedFileWebApiClient(steamHttpClient);
         this.steamUtilWebApiClient = new SteamUtilWebApiClient(steamHttpClient);
         this.steamPlayerWebApiClient = new SteamPlayerWebApiClient(steamHttpClient);
         this.steamRemoteStorageClient = new SteamRemoteStorageClient(steamHttpClient);
         this.steamNewsWebApiClient = new SteamNewsWebApiClient(steamHttpClient);
     }
 
     /**
-     * Gets the {@link SteamWorkshopWebApiClient}
-     * @return the {@link SteamWorkshopWebApiClient}
+     * Gets the {@link SteamPublishedFileWebApiClient}
+     * @return the {@link SteamPublishedFileWebApiClient}
      */
-    public SteamWorkshopWebApiClient getWorkshopWebApiClient()
+    public SteamPublishedFileWebApiClient getWorkshopWebApiClient()
```

Comment:
Do you change also getter name or it should stay as it is?

Reply:
Ah, right. I will change it.

Reply:
> Owner Author

Just askin' :P